js/index.js

```diff
@@ -1,7 +1,91 @@
-// Iteration 1: Names and Input
+console.log("hello");
 
+// Iteration 1 Names and Input 
 
-// Iteration 2: Conditionals
+let hacker1 = "Benjamin";
 
+console.log(`drivers name is ${hacker1}`);
 
-// Iteration 3: Loops
\ No newline at end of file
+let hacker2 = "Maria";
+
+console.log(`navigators name is ${hacker2}`);
+
+// Iteration 2 Conditionals 
+
+if (hacker1.length > hacker2.length) {
+    console.log(`the driver hast the longest name, it has ${hacker1.length} characters`);
+} else if (hacker1.length < hacker2.length) {
+    console.log(`the navigator hast the longest name, it has ${hacker2.length} characters`);
+} else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names ${hacker2.length} characters!`);
+}
```

Comment:
nicely done. You could have also saved some code by using an `else{}`instead of `else if` in the last condition :)